homeassistant/components/zha/core/const.py

```diff
@@ -157,20 +157,23 @@ class RadioType(enum.Enum):
     """Possible options for radio type."""
 
     ezsp = (
-        "ESZP: HUSBZB-1, Elelabs, Telegesis, Silabs EmberZNet protocol",
+        "EZSP = Silicon Labs EmberZNet protocol (ex; Elelabs, HUSBZB-1, Telegesis)",
         bellows.zigbee.application.ControllerApplication,
     )
     deconz = (
-        "Conbee, Conbee II, RaspBee radios from dresden elektronik",
+        "deCONZ = dresden elektronik deCONZ protocol (ex; ConBee I/II, RaspBee I/II)",
         zigpy_deconz.zigbee.application.ControllerApplication,
     )
     ti_cc = (
-        "TI_CC: CC2531, CC2530, CC2652R, CC1352 etc, Texas Instruments ZNP protocol",
+        "TI_CC = : Texas Instruments Z-Stack ZNP protocol (ex; CC253x, CC26x2, CC13x2)",
         zigpy_cc.zigbee.application.ControllerApplication,
     )
-    zigate = "ZiGate Radio", zigpy_zigate.zigbee.application.ControllerApplication
+    zigate = (
+        "ZiGate = ZiGate Serial protocol (ZiGate WiFi, ZiGate USB-TTL, PiZiGate)",
```

Comment:
```suggestion
        "ZiGate = ZiGate Serial protocol: PiZiGate, ZiGate USB-TTL, ZiGate WiFi",
```
let's just have them in alphabetical order

Reply:
Also, this is for radio type, do we necessarily have to include "serial protocol" words? `ZiGate Radio` won't suffice?